### PR TITLE
ENHANCE: Add unit code for each unit in units drop down menu

### PR DESCRIPTION
**Pull request by Reubsinit**
Added the unit codes for all units that appear within the units dropdown menu in the navigation bar.

### CODE REVIEW

File: src/app/common/header/header.tpl.html [hunk 1]
```diff
@@ -30,12 +30,12 @@ <h1 class="logo">Doubtfire</h1>
...
           <li ng-hide="unitRoles.length == 0" class="dropdown-header">Units You Teach</li>
           <li ng-repeat="unitRole in unitRoles | filter:isUniqueUnitRole">
-            <a href="#/units?unitRole={{unitRole.id}}">{{unitRole.unit_name}}</a>
+            <a href="#/units?unitRole={{unitRole.id}}">{{unitRole.unit_code}} - {{unitRole.unit_name}}</a>
           </li>
           <li ng-hide="unitRoles.length == 0 || projects.length == 0" class="divider"></li>
           <li ng-hide="projects.length == 0" class="dropdown-header" ng-hide="{{projects.length}}">Units You Study</li>
           <li ng-repeat="project in projects">
-            <a href="#/projects/{{project.project_id}}">{{project.unit_name}}</a>
+            <a href="#/projects/{{project.project_id}}">{{project.unit_code}} - {{project.unit_name}}</a>
```
Comment (alexcu) [inline src/app/common/header/header.tpl.html]: @Reubsinit Can we add this inside a label to make it pretty?

```html
<label class="label label-info">{{project.unit-code}}</label> {{project.unit_name}}
```

Also I think @macite want's the date too? Was that right?

**Comment by Reubsinit** [comment]
Sure thang @alexcu

**Comment by macite** [comment]
Also add in the month from the unit start date @Reubsinit. For example, I have Intro Prog from many semester so it would be good to see "COS10009 - Introduction to Programming - Jan-2016"... though that is getting quite long.

Ideas @alexcu @Reubsinit?

**Comment by Reubsinit** [comment]
@macite I'll add the month in and have a look to see if it looks to squished, possibly see what @alexcu thinks of how it looks.

**Comment by Reubsinit** [comment]
@alexcu, I can remove that .bundle. It's in my `.gitignore` as you saw the other day, but for some reason, it is pushing? I'm positive it's just a knowledge gap of mine in regards to git. Also, when I commit and push it, do I need a message?

**Pull request closed without merging**